modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -125,39 +129,67 @@ DEFAULT pxeboot
-the `coreos.inst.image_url` parameter value is the location of the compressed
-metal RAW image, and the `coreos.inst.ignition_url` parameter value is the
-location of the bootstrap Ignition config file.
+HTTP or TFTP server. The `initrd` parameter value is the location of the `initramfs`
+file on your TFTP server. The `coreos.inst.image_url` parameter value is the
+location of the compressed metal RAW image on your HTTP server, and the
+`coreos.inst.ignition_url` parameter value is the location of the bootstrap
+Ignition config file on your HTTP server.
 
 ifndef::only-pxe[]
 ** For iPXE:
 +
 ----
-kernel  http://<HTTP_server>/rhcos-<version>-installer-kernel-<architecture> ip=dhcp rd.neednet=1 initrd=http://<HTTP_server>/rhcos-<version>-installer-initramfs.<architecture>.img console=tty0 console=ttyS0 coreos.inst=yes coreos.inst.install_dev=sda coreos.inst.image_url=http://<HTTP_server>/rhcos-<version>-metal.<arhcitectutre>.raw.gz coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <1> <2>
-initrd http://<HTTP_server>/rhcos-<version>-installer-initramfs.<architecture>.img <3>
+kernel  http://<HTTP_server>/rhcos-<version>-<architecture>-installer-kernel-<architecture> ip=dhcp rd.neednet=1 initrd=rhcos-<version>-<architecture>-installer-initramfs.<architecture>.img console=tty0 console=ttyS0 coreos.inst=yes coreos.inst.install_dev=sda coreos.inst.image_url=http://<HTTP_server>/rhcos-<version>-<architecture>-metal.<architecture>.raw.gz coreos.inst.ignition_url=http://<HTTP_server>/bootstrap.ign <1> <2>
+initrd http://<HTTP_server>/rhcos-<version>-<architecture>-installer-initramfs.<architecture>.img <3>
 boot
 ----
 <1> Specify locations of the {op-system} files that you uploaded to your
 HTTP server. The `kernel` parameter value is the location of the `kernel` file,
-the `initrd` parameter value is the location of the `initramfs` file,
-the `coreos.inst.image_url` parameter value is the location of the compressed
-metal RAW image, and the `coreos.inst.ignition_url` parameter value is the
-location of the bootstrap Ignition config file.
+the `initrd` parameter value references the name of the `initramfs` file that is
+supplied on the `initrd` line below, the `coreos.inst.image_url` parameter value
+is the location of the compressed metal RAW image, and the `coreos.inst.ignition_url`
+parameter value is the location of the bootstrap Ignition config file.
 <2> If you use multiple NICs, specify a single interface in the `ip` option.
 For example, to use DHCP on a NIC that is named `eno1`, set `ip=eno1:dhcp`.
 <3> Specify the location of the `initramfs` file that you uploaded to your HTTP
 server.
 endif::only-pxe[]
 
+. If you use UEFI, perform the following actions:
+.. Provide the EFI binaries and `grub.cfg` file that are required for booting the system. You need the `shim.efi` binary and the `grubx64.efi` binary. Use one of the following methods to obtain these files:
```

Comment:
The existing PXE/iPXE instructions assume the DHCP server is already configured to point to PXELINUX or iPXE, which might be okay because setup instructions are widely available.  Is it similarly safe to assume that the customer knows how to point DHCP to shim?

Reply:
We could add some language to the prerequisites, etc that says something like "These instructions assume you are familiar with configuring DHCP/PXE infrastructure."

Reply:
Pushed an update with an edit to the prerequisite section